examples/pigweed-app/efr32/include/CHIPProjectConfig.h

```diff
@@ -120,3 +120,43 @@
  * A size, in bytes, of the individual debug event logging buffer.
  */
 #define CHIP_DEVICE_CONFIG_EVENT_LOGGING_DEBUG_BUFFER_SIZE (512)
+
+/**
+ * CHIP_DEVICE_CONFIG_BLE_FAST_ADVERTISING_INTERVAL
+ *
+ * The interval (in units of 0.625ms) at which the device will send BLE advertisements while
+ * in fast advertising mode.
+ *
+ * 40 (25ms).
+ */
+#define CHIP_DEVICE_CONFIG_BLE_FAST_ADVERTISING_INTERVAL 40
```

Comment:
I know you're just aligning the example with the other EFR32 examples, but note that these constants have been replaced with `_MIN` and `_MAX` pair since most Bluetooth APIs take a range rather than a single value. Also, if EFR32 needs custom values I suggest putting them in `src/platform/EFR32/CHIPDevicePlatformConfig.h` so that they're applied to all EFR32 examples. 
FYI @jmartinez-silabs

Reply:
@Damian-Nordic I was using it for both min max interval, I can move to using the constant _MIN _MAX you are referring to. My only thought on sharing those defines is that is the interval is in units of 0.625ms for EFR. If i remeber correctly it is the same for Nordic. I don't know for others.

Reply:
I think all platforms are expected to use the same unit as 0.625 is mentioned in the comment near to the constant definitions. In case some platform API requires a different unit, translation must be done in the platform code.